Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj

```diff
@@ -11,7 +11,6 @@
   <ItemGroup>
     <PackageReference Include="RhinoCommon" IncludeAssets="compile;build" VersionOverride="7.13.21348.13001" />
     <PackageReference Include="RhinoWindows" IncludeAssets="compile;build" VersionOverride="7.13.21348.13001" />
-    <PackageReference Include="System.Resources.Extensions" />
```

Comment:
I guess you are right, previously we were using some `.resx`. Just checkin that this library were there bc of this reason?

Reply:
I don't think we need this at all....there are no resx files.

Reply:
nice